dali/kernels/slice/slice_gpu.cuh

```diff
@@ -256,7 +256,7 @@ class SliceGPU {
     for (int i = 0; i < in.size(); i++) {
       if (default_fill_values_) {
         assert(nfill_values_ == 1);
-        fill_values_cpu[i] = static_cast<OutputType>(0.f);
+        fill_values_cpu[i] = OutputType{};
```

Comment:
Can you check if this works as intended? When I was writing the GaussianBlur, the compiler did some weird things if the output was half, and I had to initialize the 0 by using the conversion from float (that one worked without problems).